app/notifications/validators.py

```diff
@@ -218,6 +224,8 @@ def validate_template(template_id, personalisation, service, notification_type):
     if template.template_type == SMS_TYPE:
         check_sms_content_char_count(template_with_content.content_count)
 
+    check_content_is_not_blank(template_with_content)
```

Comment:
If I get this right, the `validate_template` method will be called when a request is sent through `post_notification`, right? One thing I am unsure about is if `post_bulk` will eventually call `validate_template`. I don't see it from the code. It's outside the scope of this PR though but it would be good to be proactive.

Reply:
Yeah, I looked a little bit into that but `bulk` does very different validation as far as I can tell. the file gets validated by whether it has the right columns, essentially. This is not to say that we couldn't add our row-level `is_blank` validation, but I think it deserves its own PR.

Reply:
It might have been an area that we forgot to cover.. a validation check present for atomic notification save but not bulk.